Xamarin.Forms.Controls/GalleryPages/SwipeViewGalleries/SwipeCollectionViewGallery.xaml

```diff
@@ -0,0 +1,100 @@
+                                Command="{Binding Source={x:Reference SwipeCollectionView}, Path=BindingContext.FavouriteCommand}"/>
+                        </SwipeItems>
+                    </SwipeView.LeftItems>
+                    <SwipeView.RightItems>
+                            <SwipeItems
+                                Mode="Execute">
+                                <SwipeItem
+                                    Text="Delete"
+                                    Icon="coffee.png"
+                                    BackgroundColor="Red"
+                                    Command="{Binding Source={x:Reference SwipeCollectionView}, Path=BindingContext.DeleteCommand}"/>
+                            </SwipeItems>
+                        </SwipeView.RightItems>
+                    <SwipeView.Content>
+                        <Grid
+                            BackgroundColor="White"
+                            HeightRequest="80"
+                            RowSpacing="0">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="Auto" />
+                            </Grid.ColumnDefinitions>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="*" />
+                            </Grid.RowDefinitions>
+                            <Label
+                                Grid.Column="0"
+                                Grid.Row="0"
+                                Text="{Binding Title}"
+                                Style="{StaticResource TitleStyle}"/>
+                            <Label
+                                Grid.Column="1"
+                                Grid.Row="0"
+                                Text="{Binding Date}"
+                                Style="{StaticResource DateStyle}"/>
+                            <Label
+                                Grid.Column="0"
+                                Grid.ColumnSpan="2"
+                                Grid.Row="1"
+                                Text="{Binding SubTitle}"
+                                Style="{StaticResource SubTitleStyle}"/>
+                            <Label
+                                Grid.Column="0"
+                                Grid.ColumnSpan="2"
+                                Grid.Row="2"
+                                Text="{Binding Description}"
+                                Style="{StaticResource SubTitleStyle}"/>
+                        </Grid>
+                    </SwipeView.Content>
+                </SwipeView>
+            </DataTemplate>
+
+        </ResourceDictionary>
+    </ContentPage.Resources>
+    <ContentPage.Content>
+        <Grid>
+            <CollectionView
```

Comment:
This doesn't look like it's playing well with recycling on Android. If you swipe open an item then scroll down you'll see a bunch of Cells now have an open SwipeView

Reply:
I have reviewed this case and you are right. I have tested SwipeContainer in UWP and I have applied the same logic. If the scroll start and the SwipeView is open, close it.
![scroll-swipeview](https://user-images.githubusercontent.com/6755973/65589324-23769b80-df89-11e9-80a7-c4de2e95ef94.gif)